src/components/NodeBlock.tsx

```diff
@@ -106,30 +106,32 @@ export function NodeBlock({
         <code id="node-type" className="text-purple">
           {nodeType}
         </code>
-        <div
-          onClick={(e) => {
-            const value =
-              e.currentTarget.children.namedItem('node-value')?.textContent;
-            void navigator.clipboard.writeText(value ?? '');
+        {state.value !== 'undefined' && (
+          <div
+            onClick={(e) => {
+              const value =
+                e.currentTarget.children.namedItem('node-value')?.textContent;
+              void navigator.clipboard.writeText(value ?? '');
 
-            setCopyNotification(true);
-            setTimeout(() => {
-              setCopyNotification(false);
-            }, 1500);
-          }}
-          className="flex h-10 items-center gap-3 rounded-md border 
+              setCopyNotification(true);
```

Comment:
Why are there new spaces here? Would make sense if the whole block would be to the right because of the new condition, but here it is only partially shifted

Reply:
Linter passed so it's probably just a Github misunderstanding of what happened or smth.